src/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,116 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+import styled from 'styled-components';
+import { text, color } from '@storybook/addon-knobs';
+// ANCHOR
+import { Typography } from '..';
+import { colors } from '../theme';
+import * as README from './README.md';
+import SomeImage from './temporary/some_image.svg';
+import { Grid, CustomAdaptor, StandardBreakpoints } from '../Grid';
+import { Button } from '../Button';
+
+import { Hero } from './Hero.component';
+const { Title, Subtitle } = Hero;
+
+// Styled Components
+// ------------------------------------------------------------------------------------------------------------------
+const StyledStory = styled.div`
+    background: ${colors.white.base};
+    width: 100vw;
+`;
+
+// Stories
+// ------------------------------------------------------------------------------------------------------------------
+const tealGradient = 'linear-gradient(224deg,#00cccd,#2de2b3)';
```

Comment:
Few questions: 
- Are these standard colors?
- Do they exist in our `colors`?
    - What's the case for adding them?

Reply:
I ripped this from the rmn banner, and didn't worry too much about having them being standard because this is just a backdrop for some stories.

It appears `#00cccd` is `tealBreaker.base`, and I don't think `#2de2b3` is standard.

Reply:
`dealEnvy.light` is pretty similar and in the new mockups I see, so I'll just switch to that.